cfme/configure/tasks.py

```diff
@@ -147,61 +149,110 @@ def wait_analysis_finished(task_name, task_type, delay=5, timeout='5M'):
              delay=delay, timeout=timeout, fail_func=requests.reload)
 
 
+class TasksView(BaseLoggedInPage):
+    flash = FlashMessages('.//div[starts-with(@id, "flash_text_div")]')
+    # Toolbar
+    delete = Dropdown('Delete Tasks')  # dropdown just has icon, use element title
+    reload = Button(title='Reload the current display')
+
+    @View.nested
+    class tabs(View):  # noqa
+        # Extra Toolbar
+        # Only on 'All' type tabs, but for access it doesn't make sense to access the tab for a
+        # toolbar button
+        cancel = Button(title='Cancel the selected task')
+
+        # Form Buttons
+        apply = Button('Apply')
+        reset = Button('Reset')
+        default = Button('Default')
+
+        # Filters
+        zone = BootstrapSelect(id='chosen_zone')
+        period = BootstrapSelect(id='time_period')
+        user = BootstrapSelect(id='user_choice')
+        # This checkbox search_root captures all the filter options
+        # It will break for status if/when there is second checkbox selection field added
+        # It's the lowest level div with an id that captures the status checkboxes
+        status = CheckboxSelect(search_root='tasks_options_div')
+        state = BootstrapSelect(id='state_choice')
+
+        @View.nested
+        class mytasks(Tab):  # noqa
+            TAB_NAME = "My VM and Container Analysis Tasks"
+            table = Table(table_loc)
+
+        @View.nested
+        class myothertasks(Tab):  # noqa
```

Comment:
OPTIONAL: @mshriver , do you mind if I ask about your opinion on absent underscores ?